src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -31,10 +33,11 @@ public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
 
     }
 
-    public CommandResult(String feedbackToUser, boolean isDisplayCommand) {
+    public CommandResult(String feedbackToUser, Person firstMatchedPerson, boolean isDisplayCommand) {
         this.feedbackToUser = requireNonNull(feedbackToUser);
         this.isDisplayCommand = isDisplayCommand;
         this.showHelp = false;
```

Comment:
Its a bit of a nitpick, but it would be more readable if you have one main constructor with all the fields. Currently the person object is not initialised to anything otherwise explicitly when one of the other constructors are called.

so have one constructor CommandResult(String feedback, Person person, boolean, boolean, boolean) 
then your 2 other constructors call this constructor with different parameters. 

also requireNonNull for the person object for the CommandResult constructor used by display command